lms/static/js/spec/student_profile/learner_profile_fields_spec.js

```diff
@@ -0,0 +1,276 @@
+              MessageBannerView) {
+        'use strict';
+
+        describe("edx.user.LearnerProfileFields", function () {
+
+            var createImageView = function (ownProfile, hasImage, imageMaxBytes, imageMinBytes, yearOfBirth) {
+
+                var imageData = {
+                    image_url_large: '/media/profile-images/default.jpg',
+                    has_image: hasImage ? true : false
+                };
+
+                yearOfBirth = _.isUndefined(yearOfBirth) ? 1989 : yearOfBirth;
+
+                var accountSettingsModel = new UserAccountModel();
+                accountSettingsModel.set({'profile_image': imageData});
+                accountSettingsModel.set({'year_of_birth': yearOfBirth});
+                accountSettingsModel.set({'requires_parental_consent': _.isEmpty(yearOfBirth) ? true : false});
+
+                accountSettingsModel.url = Helpers.USER_ACCOUNTS_API_URL;
+
+                var messageView = new MessageBannerView({
+                    el: $('.message-banner')
+                });
+
+                imageMaxBytes = imageMaxBytes || 64;
+                imageMinBytes = imageMinBytes || 16;
+
+                var editable = ownProfile ? 'toggle' : 'never';
+
+                return new LearnerProfileFields.ProfileImageFieldView({
+                    model: accountSettingsModel,
+                    valueAttribute: "profile_image",
+                    editable: editable === 'toggle',
+                    messageView: messageView,
+                    imageMaxBytes: imageMaxBytes,
+                    imageMinBytes: imageMinBytes,
+                    imageUploadUrl: Helpers.IMAGE_UPLOAD_API_URL,
+                    imageRemoveUrl: Helpers.IMAGE_REMOVE_API_URL
+                });
+            };
+
+            beforeEach(function () {
+                setFixtures('<div class="message-banner"></div><div class="wrapper-profile"><div class="ui-loading-indicator"><p><span class="spin"><i class="icon fa fa-refresh"></i></span> <span class="copy">Loading</span></p></div><div class="ui-loading-error is-hidden"><i class="fa fa-exclamation-triangle message-error" aria-hidden=true></i><span class="copy">An error occurred. Please reload the page.</span></div></div>');
+                TemplateHelpers.installTemplate('templates/student_profile/learner_profile');
+                TemplateHelpers.installTemplate('templates/fields/field_image');
+                TemplateHelpers.installTemplate("templates/fields/message_banner");
+            });
+
+            var createFakeImageFile = function (size) {
+                var fileFakeData = 'i63ljc6giwoskyb9x5sw0169bdcmcxr3cdz8boqv0lik971972cmd6yknvcxr5sw0nvc169bdcmcxsdf';
+                return new Blob(
+                    [ fileFakeData.substr(0, size) ],
+                    { type: 'image/jpg' }
+                );
+            };
+
+            it("can upload profile image", function() {
```

Comment:
Nit: it would be good to have all of these inside a describe as follows:

    describe("ProfileImageFieldView", function () {

Then the test names will read cleanly, and also all the profile image tests can be run together.